module/VuFind/src/VuFind/Role/PermissionProvider/User.php

```diff
@@ -29,8 +29,9 @@
 
 namespace VuFind\Role\PermissionProvider;
 
-use LmcRbacMvc\Service\AuthorizationService;
+use VuFind\View\Helper\Root\Auth;
```

Comment:
Sorry, I thought this class was using \VuFind\Auth\Manager rather than \LmcRbacMvc\Service\AuthorizationService, so this was a more complicated change than I had realized.

It would probably be better to use \VuFind\Auth\Manager instead of \VuFind\View\Helper\Root\Auth, which is just a view helper that wraps around the auth manager.

But maybe for now the best thing is to revert this part of the change -- in order to make it work, we'll have to create a new custom factory and make some other changes, and none of that is necessary for the Doctrine work you are actively doing. Probably better to consider it out of scope for now and handle it separately.

Sorry again for leading you down the wrong path! :-)